src/components/HeroProjectContainer.js

```diff
@@ -18,14 +22,23 @@ function HeroProjectContainer({styles, heroProject}) {
     <div className={styles['hero-project-container-div']}>
       <div className={styles['hero-project-image-div']}>
         <img src={images[0].link} />
-        <p>click for more images</p>
+        <p onClick={() => setIsViewerOpen(!isViewerOpen)}>click for more images</p>
```

Comment:
I would use a `<button>` element here. It's better for accessibility and it comes with some freebees (like not requiring keyboard events) that make it a bit nicer to use.

Reply:
Totally agree. I've switched it over and it will be included in the next PR